scripts/python/list-contributors.py

```diff
@@ -201,10 +201,12 @@ def get_contributors() -> Dict[str, List]:
 
     for node in nodes:
         pr_num = node["number"]
-        author = node["author"]["login"]
-        reviewers = {r["author"]["login"] for r in node["reviews"]["nodes"]} - {author}
-        commenters = {r["author"]["login"] for r in node["comments"]["nodes"]} - {author}
-
+        try:
+            author = node["author"]["login"]
+            reviewers = {r["author"]["login"] for r in node["reviews"]["nodes"]} - {author}
+            commenters = {r["author"]["login"] for r in node["comments"]["nodes"]} - {author}
+        except:
+            continue
```

Comment:
maybe log a warning?

Reply:
This should be fine for now. I don't plan to spend more time on updating this script.